language/stdlib/modules/Testnet.move

```diff
@@ -21,5 +21,97 @@ module Testnet {
         Transaction::assert(Signer::address_of(account) == 0xA550C18, 0);
         IsTestnet{} = move_from<IsTestnet>(0xA550C18);
     }
+
+    // **************** SPECIFICATIONS ****************
+
+    /*
```

Comment:
Is this intended to be part of the documentation? Then one should use `/**`.

Also if this is general information, it could as well go to the module, and doesn't belong into the specification section.